app/controllers/insured/plan_shoppings_controller.rb

```diff
@@ -290,6 +293,27 @@ def agreed_to_thankyou_ivl_page_terms
     false
   end
 
+  def set_cache_headers
+    response.headers["Cache-Control"] = "no-cache, no-store, private, must-revalidate"
```

Comment:
Reasons for using this Cache-Control header are:

1.  The page will not be cached.
2.   The must-revalidate directive ensures that, in general, the browser's back button will not reuse the cached page, but will instead make a call to the server.